manifests/default_mods.pp

```diff
@@ -16,7 +16,7 @@
       if $facts['os']['name'] != 'Amazon' and $use_systemd {
         ::apache::mod { 'systemd': }
       }
-      if ($facts['os']['name'] == 'Amazon' and $facts['os']['release']['full'] == '2') {
+      if ($facts['os']['name'] == 'Amazon' and $facts['os']['release']['major'] =~ /^(2|2023)$/) {
```

Comment:
does it make more sense to always include it when systemd is the default init system?

Reply:
Yeah probably @bastelfreak 
I think the original logic here was added to support AL1 & 2, with the former not using systemd. But since that's been EOL for a while now, the Amazon specific code shouldn't really be needed anymore.

Reply:
I went ahead and updated the PR.